src/Stryker.Core/Stryker.Core/MutantFilters/BroadcastMutantFilter.cs

```diff
@@ -8,42 +8,33 @@ namespace Stryker.Core.MutantFilters
 {
     public class BroadcastMutantFilter : IMutantFilter
     {
+        public MutantFilter Type => MutantFilter.Broadcast;
```

Comment:
Why do the filters need a Type now?

Reply:
So we can use their index as the sort value (enums are ints so their position decides their order in a sorted list), this guarantees the order they are executed on.

Reply:
Maybe we should add a comment to the enum to make sure nobody changes the order :)

Reply:
Yep good idea

Reply:
The comment is added to the MutantFilter enum